### PR TITLE
Updated cp behavior

### PR DESCRIPTION
**Pull request by SamMarkowitz**

### CODE REVIEW

File: docs/12_lesson.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ Then, all we really need to do is create a task in our flow that will call the `
 ##Run It
 We can save the files, run the flow and check that an email was sent with the proper information.
 ```bash
-run --f <folder path>/tutorials/hiring/new_hire.sl --cp <folder path>/tutorials/base,<folder path>/tutorials/hiring,<content folder path>/io/cloudslang/base --i first_name=john,last_name=doe
+run --f <folder path>/tutorials/hiring/new_hire.sl --cp <folder path>/tutorials/base,<folder path>/tutorials/hiring --i first_name=john,last_name=doe
```
Comment (oritstone) [inline docs/12_lesson.md]: Are you sure you can remove it? Is it working this way?
@orius123 's fix is that if you mention any cp, you need to specify also the base content (if needed)

File: docs/13_lesson.md [hunk 2]
```diff
@@ -86,7 +86,7 @@ Notice that in this case we're using the system properties on the sending end, r
 We can save the files and run the flow to see that the values are being taken from the system properties file we specify. If we want to swap out the values with another set, all we have to do is point to a different system properties file.
 
 ```bash
-run --f <folder path>/tutorials/hiring/new_hire.sl --cp <folder path>/tutorials/base,<folder path>/tutorials/hiring,<content folder path>/base --i first_name=john,last_name=doe --spf <folder path>/tutorials/properties/bcompany.yaml
+run --f <folder path>/tutorials/hiring/new_hire.sl --cp <folder path>/tutorials/base,<folder path>/tutorials/hiring --i first_name=john,last_name=doe --spf <folder path>/tutorials/properties/bcompany.yaml
```
Comment (oritstone) [inline docs/13_lesson.md]: same comment as above

**Comment by oritstone** [comment]
same comment for all fixes as above

File: docs/12_lesson.md (hunk 1 above)
Comment (SamMarkowitz) [inline docs/12_lesson.md]: You're right. I misunderstood it.

**Comment by SamMarkowitz** [comment]
Was correct as it. Closing PR.

**Pull request closed without merging**